jsonb/src/main/java/io/avaje/jsonb/stream/JGenerator.java

```diff
@@ -37,7 +37,7 @@ final class JGenerator implements JsonGenerator {
   private static final byte[] NULL = "null".getBytes(StandardCharsets.UTF_8);
   private static final byte[] TRUE = "true".getBytes(StandardCharsets.UTF_8);
   private static final byte[] FALSE = "false".getBytes(StandardCharsets.UTF_8);
-  private static final byte[] INDENT = "  ".getBytes(StandardCharsets.UTF_8);
+  private static final byte[] INDENT = "    ".getBytes(StandardCharsets.UTF_8);
```

Comment:
Why push this out to 4?

Reply:
that's how my ide formats json so I just went with it

Reply:
you can change if you like

Reply:
Yeah, it's in the area of "breaking change" for any asserting on json strings (which is a little risky I know). I'll put it back for now and keep pondering ...